impl-base/src/main/java/org/jboss/shrinkwrap/impl/base/URLPackageScanner.java

```diff
@@ -140,10 +148,10 @@ private void handleArchiveByFile(File file) throws IOException, ClassNotFoundExc
             while (entries.hasMoreElements()) {
                 ZipEntry entry = entries.nextElement();
                 String name = entry.getName();
-                if (name.startsWith(packageNamePath) && name.endsWith(".class")
-                    && (addRecursively || !name.substring(packageNamePath.length() + 1).contains("/"))) {
-                    String className = name.replace("/", ".").substring(0, name.length() - ".class".length());
-                    foundClass(className);
+                if (name.startsWith(prefix + packageNamePath) && name.endsWith(".class")
+                        && (addRecursively || !name.substring((prefix + packageNamePath).length() + 1).contains("/"))) {
+                    String className = name.replace("/", ".").substring(prefix.length(), name.length() - ".class".length());
```

Comment:
I'll fix this up in the AM

--
Sent from my iPhone


> On May 31, 2016, at 5:32 PM, Andrew Lee Rubinger <notifications@github.com> wrote:
> 
> In impl-base/src/main/java/org/jboss/shrinkwrap/impl/base/URLPackageScanner.java:
> 
> > @@ -140,10 +148,10 @@ private void handleArchiveByFile(File file) throws IOException, ClassNotFoundExc
> >              while (entries.hasMoreElements()) {
> >                  ZipEntry entry = entries.nextElement();
> >                  String name = entry.getName();
> > -                if (name.startsWith(packageNamePath) && name.endsWith(".class")
> > -                    && (addRecursively || !name.substring(packageNamePath.length() + 1).contains("/"))) {
> > -                    String className = name.replace("/", ".").substring(0, name.length() - ".class".length());
> > -                    foundClass(className);
> > +                if (name.startsWith(prefix + packageNamePath) && name.endsWith(".class")
> > +                        && (addRecursively || !name.substring((prefix + packageNamePath).length() + 1).contains("/"))) {
> > +                    String className = name.replace("/", ".").substring(prefix.length(), name.length() - ".class".length());
> Nit: externalize ".class" here and below. ALR will handle in rebase.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>